packages/cinterop/build.gradle.kts

```diff
@@ -318,22 +342,47 @@ android {
 
 // Building Mach-O universal binary with 2 architectures: [x86_64] [arm64] (Apple M1) for macOS
 val capiMacosUniversal by tasks.registering {
-    build_C_API_Macos_Universal(releaseBuild = isReleaseBuild)
+    build_C_API_Macos_Universal(buildType = buildType)
```

Comment:
`buildType` name shadowing with https://github.com/realm/realm-kotlin/blob/09c10bad10060635674abb40b697e0e58e631c12/packages/cinterop/build.gradle.kts#L53 :) maybe rename the global one